### PR TITLE
Update to warn against Function constructor

### PR DESCRIPTION
**Pull request by Ryan-McBride**
Added section 7.9 to warn against using the Function creator, as it opens similar vulnerabilities as eval.
Originally suggested in pull request #395.

No other lines were edited or removed.

screen shot 2015-07-04 at 10 19 36 am

### CODE REVIEW

**Comment by chrisngobanh** [comment]
Nitpick: Section 6.1 says to use single quotes instead of double quotes for strings.

**Comment by Ryan-McBride** [comment]
Good call, fixed.

File: README.md [hunk 1]
```diff
@@ -567,6 +567,14 @@
   count();  // 3
   ```
 
+- [7.9](#7.9) <a name='7.9'></a> Never use the Function constructor to create a new function.
+
+  > Why? Creating a function in this way evaluates a string similarly to eval(), which opens vulnerabilities.
+
+  ```javascript
+  // bad
+  var add = new Function('a', 'b', 'return a + b');
```
Comment (ljharb) [inline README.md]: Also `var subtract = Function('a', 'b', 'return a - b');` is bad also.

Comment (Ryan-McBride) [inline README.md]: Added to this PR.

**Comment by goatslacker** [comment]
Thanks!

**Pull request merged**